conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+                
+            <p>Everyone involved in the conflict resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
+            </ul>
+                
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+
+        <section id="procedures">
+            <h2>Conflict Resolution Procedures</h2>
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>
+                    </ul>
+                </li>
+                <li>Incident is discussed, impact is shared, and first resolution is proposed.
+                    <ul>
+                        <li>For conflicts that have a clear path to resolution and low severity (for example, a harsh criticism in a meeting or inappropriate language), the focus should be on coming to a shared understanding of what happened and agreeing to a path to resolution (for example, an apology or editing a comment or minutes).</li>
+                    </ul>  
+                </li>
+                <li>
+                    ...
+                </li>
+            </ol>
+        </section>
+
+        <section id="ombudspeople">
+            <h2>The Role of the Ombudsperson</h2>
+
+            <p>Ombudspeople are advisors to participants who can provide guidance, advice, and support during the conflict resolution process. Ombudspeople are meant to be neutral third parties to any incident, and as such are required not to take sides or provide opinions on the indicidents they are involved in.</p>
```

Comment:
I certainly agree with the need for Ombuds to be neutral.  But do we really want Ombuds to not provide opinions on the incidents that are reported to them?  I think this last sentence needs more consideration.  It seems to me that:

- It will be nearly impossible to prevent an Ombud from expressing an opinion -- especially since it is very important for an Ombud to be empathetic; and
- An Ombud's professional opinion could be enormously helpful to those who are involved in an incident.

Also, spelling error: s/indicidents/incidents/